client/components/Planning/FeaturedPlanning/FeaturedPlanningSelectedList.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FeaturedPlanningList} from './FeaturedPlanningList';
+import './style.scss';
+
+export const FeaturedPlanningSelectedList = ({
+    items,
+    readOnly,
+    lockedItems,
+    currentSearchDate,
+    dateFormat,
+    timeFormat,
+    activeFilter,
+    filter,
+    loadingIndicator,
+    desks,
+    users,
+    leftBorder,
+    onRemoveFromSelectedFeaturedPlanning,
+    selectedPlanningIds,
+    onSortEnd,
+    onSortStart,
+    highlights,
+    onClick,
+}) => (
+    <FeaturedPlanningList
+        items={items}
+        readOnly={readOnly}
+        lockedItems={lockedItems}
+        currentSearchDate={currentSearchDate}
+        dateFormat={dateFormat}
+        timeFormat={timeFormat}
+        activeFilter={activeFilter}
+        filter={filter}
+        loadingIndicator={loadingIndicator}
+        desks={desks}
+        users={users}
+        onRemoveFromSelectedFeaturedPlanning={onRemoveFromSelectedFeaturedPlanning}
+        selectedPlanningIds={selectedPlanningIds}
+        hideItemActions
+        onSortEnd={onSortEnd}
+        onSortStart={onSortStart}
+        sortable={!readOnly}
+        leftBorder={leftBorder}
+        header={gettext(`Featured stories for ${currentSearchDate.format('dddd LL')}`)}
```

Comment:
This should use gettext with params instead of injecting the value in the string that is passed to gettext.
i.e. `getttext('Featured stores for {{ date }}', {date: currentSearchDate.format('dddd LL')})`